it/src/test/java/org/eclipse/jkube/integrationtests/springboot/zeroconfig/ZeroConfigK8sGradleITCase.java

```diff
@@ -132,7 +132,7 @@ void k8sHelm() {
     assertThat(helmDirectory.resolve(getApplication() + "-0.0.0-SNAPSHOT.tar.gz").toFile(),
       anExistingFile());
     assertThat(helmDirectory.resolve("Chart.yaml").toFile(), yaml(allOf(
-      aMapWithSize(3),
+      aMapWithSize(5),
```

Comment:
Why has map size increased by 2 in this case?

Reply:
typo